resolver/context.go

```diff
@@ -12,6 +12,7 @@ type context struct {
 	defIdx       int
 	depIdx       int
 	argIdx       int
+	errDetailIdx int
```

Comment:
Please add `errDetailIdx` field to [`clone()`](https://github.com/mercari/grpc-federation/pull/68/files#diff-2404bd4fa6ff38bb60e2e9c2e51a79c8f5a5d6a9135ae29ca6bf393d6d875020R31) method